sig/rake.rbs

```diff
@@ -1,6 +0,0 @@
-module Rake
-  class TaskLib
```

Comment:
I also removed rake.rbs. This class has also been filed into gem_rbs_colleciton now.